jbpm-task-assigning/jbpm-task-assigning-core/src/main/java/org/jbpm/task/assigning/model/AbstractPersistable.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.task.assigning.model;
+
+import java.io.Serializable;
+
+import org.apache.commons.lang3.builder.CompareToBuilder;
+import org.optaplanner.core.api.domain.lookup.PlanningId;
+
+public abstract class AbstractPersistable implements Serializable,
+                                                     Comparable<AbstractPersistable> {
```

Comment:
A couple weeks ago, we've refactored `AbstractPersistable` to no longer be `Comparable`.
Just FYI, in case you want to keep the code in sync with `optaplanner-examples`.